cmd/podman/networks/reload.go

```diff
@@ -24,8 +24,7 @@ var (
 			return validate.CheckAllLatestAndIDFile(cmd, args, false, "")
 		},
 		ValidArgsFunction: common.AutocompleteContainers,
-		Example: `podman network reload --latest
```

Comment:
FYI network reload is not support via remote so it is technically not needed to remove it here but for consistency it properly makes sense to remove it anyway

Reply:
Man page at least clearly state that it is not supported for Remote, although Mac and Windows users probably do not think they are remote.

Reply:
Yes the description under the option which is fine but I mean the ones shown under the Examples section in the man page, some matches I can find with `grep -rn  " --latest" ./docs/source`

Reply:
Lets get this merged and then we can fix man pages with a CI:DOCS PR.